Formula/opencascade.rb

```diff
@@ -0,0 +1,91 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://dev.opencascade.org/"
+  url "https://github.com/FreeCAD/homebrew-freecad/releases/download/0/opencascade-7.2.0.tgz"
+  sha256 "f9a81ca2267635773af9fff80625735dd4f65389af986fe93a94263081b422e9"
+
+  bottle do
```

Comment:
Remove the bottle block, it will be regenerated automatically.

Reply:
Per the instructions I read, it was my understanding that I was supposed to leave the bottle block so is this suggestion specific to porting a formula from science to core?  Will fix.

Reply:
We've been inconsistent with this. It has no impact other than what message is used for the new bottles: "added" vs "updated"